1-js/07-object-oriented-programming/02-property-accessors/article.md

```diff
@@ -55,7 +55,7 @@ alert(user.fullName); // John Smith
 */!*
 ```
 
-From outside, an accessor property looks like a regular one. That's the idea of accessor properties. We don't call `user.fullName` as a function, we read it normally: the getter runs behind the scenes.
+From outside, an accessor property looks like a regular one. That's the idea of accessor properties. We don't **call** `user.fullName` as a function, we **read** it normally: the getter runs behind the scenes.
```

Comment:
Maybe use italic here _call_, not bold **call** ?

To use italic, place a single asterisk.